canonical-web-and-design/snapcraft.io
# Move promote and close channel actions to single menu

Fixes #2035

Removes close channel icon and promote button, merging both into channel settings menu.

### QA

- ./run or https://snapcraft-io-canonical-web-and-design-pr-2036.run.demo.haus/
- go to releases page of some snaps
- there should be no 'Promote' button or close channel bin icon
- click on cog settings button, menu with promote and close options should appear
- try different edge cases:
  - make sure everything that can be promoted is available in menu
  - make sure channels in devmode can't be promoted to stable/candidate
  - make sure empty/closed channels can't be promoted or closed
- try closing channels
- try promoting channels
- everything should work as before with new menu

<img width="1038" alt="Screenshot 2019-06-25 at 13 32 06" src="https://user-images.githubusercontent.com/83575/60094945-b109db80-974d-11e9-9c1f-1306248a0251.png">
